src/eos-autoupdater.c

```diff
@@ -59,7 +59,7 @@ typedef enum _UpdateStep {
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
+static const char *UPDATE_STAMP_DIR = SHAREDSTATEDIR "/eos-updater";
```

Comment:
Fair, you overrode it in debian/rules. Still, I don't understand what's wrong with `LOCALSTATEDIR "/lib/eos-updater"`. Even dpkg and apt do it this way - https://anonscm.debian.org/cgit/dpkg/dpkg.git/tree/configure.ac#n55, https://anonscm.debian.org/cgit/apt/apt.git/tree/CMakeLists.txt#n195. They both use localstatedir of `/var` in debian/rules (dpkg manually, and apt via dh_auto_configure).